_pages/cn/home.md

```diff
@@ -18,34 +18,35 @@ intro:
   - excerpt: "
 <div class='excerpt-title'>最新新闻</div>
 
+<style> li{width:100%} </style>
 <ul>
   <li>
-    <a href='/cn/release/pack-downloads'>Apache ServiceComb Pack 发布0.5.0</a>
+    <a href='/cn/release/'>Apache ServiceComb 发布正式版 1.2.0</a>
   </li>
   <li>
-    <a href='/cn/release/java-chassis'>Apache ServiceComb Java-Chassis 发布1.2.1</a>
+    <a href='/cn/release/pack-downloads'>Apache ServiceComb Pack 发布0.5.0</a>
   </li>
   <li>
-    <a href='/cn/docs/mersher-quick-start/'>Apache ServiceComb 新项目 Mesher </a>
+    <a href='/cn/release/toolkit-downloads'>Apache ServiceComb Toolkit 发布0.1.0</a>
   </li>
   <li>
-    <a href=''>Apache ServiceComb 荣获中国开源云联盟组织的中国优秀开源项目一等奖</a>
+    <a href='/cn/release/java-chassis'>Apache ServiceComb Java-Chassis 发布1.2.1</a>
```

Comment:
path of '/cn/release/java-chassis' is incorrect

Reply:
thanks, I have fixed this problem in the new commit

Reply:
Thanks, I have fixed this problem in the new commit, pls help to check